### ISSUE TITLE
blue pixel

### ISSUE THREAD

**Issue by winnieepooh** [jantic/DeOldify#461]
Hi,
I am training dataset by following the step in ColorizeTrainingArtistic.ipynb.But when the 192px training is done，The image stored in the ArtisticModel_image_gen showed a lot of blue pixels interference.Is this normal or is there a reason for this?

![ILSVRC2012_val_00001113](https://user-images.githubusercontent.com/94221441/228414265-105c58aa-b9c2-442b-b5f2-1f06cb1d915f.JPEG)
![ILSVRC2012_val_00001664](https://user-images.githubusercontent.com/94221441/228414297-a04f2c97-939a-49b5-90f9-4a854b778e03.JPEG)
Thanks!

**Comment by jantic** [comment]
That's basically the sort of thing you see when you've busted your model weights during training. Did you do anything different or are you following the training steps verbatim? I can tell you you're more likely to run into this if you're attempting to train using mixed precision.

**Comment by winnieepooh** [comment]
@jantic Thanks for your prompt reply.I only changed the batch size due to low memory.Others I'm sure that I followed the training steps verbatim.Does the small batch size affect the training results?

**Comment by jantic** [comment]
Lowering the batch size will cause this sort of issue too, actually. You could try to lower the learning rate proportionately but really I'd recommend either trying [mixed precision training ](https://docs.fast.ai/callback.fp16.html) (learner.to_fp16()) or securing a larger video card (11GB+ gpu memory. Note that I haven't tried mixed precision here on this particular project so you may or may not have success with that.

**Comment by jantic** [comment]
I should mention too- any changes in batch size should be accompanied by a proportionate change in learning rate.  For example- if 2x lower batch size, then 2x lower learning rate. Just doing that if you haven't already might address your issue but I will say that going lower than the already very low batch sizes I used here will probably not be great.
